worker/games.py

```diff
@@ -884,6 +891,9 @@ def launch_cutechess(
         stderr=subprocess.STDOUT,
         universal_newlines=True,
         bufsize=1,
+        # The next option is necessary to able be to send CTRL_C_EVENTs.
```

Comment:
typo "to be able"?

Reply:
Yep.... Luckily these are the easiest bugs to fix.